### ISSUE TITLE
Add PlutoUtils.jl without installing the `pluto` command?

### ISSUE THREAD

**Issue by fonsp** [fonsp/PlutoUtils.jl#16]
Since this is also just a collection of scripts useful for other things, maybe we should move the CLI installation to a function?
```julia
] add PlutoUtils
using PlutoUtils
install_pluto_CLI()
```
Since it's interactive, we could also prompt whether they want it to be added to their path.

What do you think? @Roger-luo

**Comment by fonsp** [comment]
Okay! Let's leave it in

**Comment by fonsp** [comment]
I want to add more functionality to this repo (static document export), so we should not install the CLI by default anymore. Can you make it an extra step @Roger-luo ?

**Comment by Roger-luo** [comment]
Yeah, I think I'll move the repo to PlutoCLI first, then rewrite this - sorry I didn't get much time to maintain it in the past few months to make it keep up with the updates from Pluto. I think it needs a rewrite now.

**Comment by fonsp** [comment]
No worries! I think it would be nice in its own repo, we can rename this repo to PlutoCLI. This means that I can't create a new one called PlutoUtils, but that's fine because it wasn't going anywhere useful :)
